src/Components/ContactForm/ContactForm.js

```diff
@@ -36,7 +31,7 @@ class ContactForm extends Component {
     // }
   };
 
-  displayErrorMessage = () => {
+  const displayErrorMessage = () => {
```

Comment:
This function currently has no functionality, but I've left the code in case I decide to show an error message. Although the best way now might just be to update state.